Source/SplineFollow/Private/Components/SFSplineComponent.cpp

```diff
@@ -128,6 +128,41 @@ void USFSplineComponent::Serialize( FArchive & archive )
 }
 
 #if WITH_EDITOR
+void USFSplineComponent::SaveSplineMarkers( TArray< FSFSplineMarker > markers_to_save )
+{
+    SplineMarkers.Reset();
+
+    const auto create_marker = [ & ]< typename U >( const auto & marker, U * object ) {
```

Comment:
I think you can remove the typename and just use UObject instead

Reply:
done